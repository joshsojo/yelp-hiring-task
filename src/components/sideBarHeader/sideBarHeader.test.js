import { shallow } from "enzyme";
import React from "react";
import SideBarHeader from "./sideBarHeader.component";

describe("Side Bar Header Component", () => {
  test("Renders without crashing", () => {
    expect(shallow(<SideBarHeader />)).toMatchSnapshot();
  });
});
