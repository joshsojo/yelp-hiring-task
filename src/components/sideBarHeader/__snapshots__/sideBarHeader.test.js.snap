// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Side Bar Header Component Renders without crashing 1`] = `ShallowWrapper {}`;
